src/overlay012/ov12_02235E94.c

```diff
@@ -134,26 +135,26 @@ static BOOL ov12_02237474(BallRotation *param0);
  * These NARC members describe the sprite, palette, cell, and anim to use for that ball.
  */
 static const int ov12_BallIndexToNARCMembers[][4] = {
```

Comment:
```suggestion
static const int sThrownBallSpriteResources[][4] = {
```

Also check if it's possible to turn this into an array of structs like these (this name is likely not unique):

```c
struct SpriteResource {
    int tiles;
    int palette;
    int cells;
    int animation;
};
```

Reply:
I might need some advice on this one, the array gets used later as `sThrownBallSpriteResources[v0][param1]`, so if I convert it to an array of struct the `[param1]` would be trying to index that struct as if it's an array

I can do something like the below and then have the array access use the array part of the union but idk if this is good style. Or is there a better alternative to make this work?

```
typedef union AnimatedSpriteResource {
    struct AnimatedSpriteResource_s {
        int tiles;
        int palette;
        int cells;
        int animation;
    };
    int AnimatedSpriteResource_a[4];
} AnimatedSpriteResource;
```